bots/tests-invoke

```diff
@@ -126,7 +126,7 @@ class PullTask(object):
 
         # build a globally unique test context for GitHub statuses
         github_context = self.context
-        if self.test_project:  # disambiguate tests for external projects
+        if self.test_project != api.repo :  # disambiguate tests for external projects
```

Comment:
The code change is right, but the comment is now misleading. How about "disambiguate test name for foreign project tests"?